nix/ocaml.nix

```diff
@@ -189,25 +216,7 @@ let
         OCAMLPARAM = "_,cclib=-lc++";
       });
 
-      mina = let
-        commit_sha1 =
-          inputs.self.sourceInfo.rev or "<unknown>                               ";
-        commit_date =
-          inputs.self.sourceInfo.lastModifiedDate or "<unknown>     ";
-      in pkgs.runCommand "mina-release" {
-        buildInputs = [ pkgs.makeWrapper ];
-        outputs = self.mina-dev.outputs;
-      } (map (output: ''
-        cp -R ${self.mina-dev.${output}} ${placeholder output}
-        chmod 700 ${placeholder output} -R
-        for i in $(find "${placeholder output}/bin" -type f); do
-          sed 's/__commit_sha1___________________________/${commit_sha1}/' -i "$i"
-          sed 's/__commit_date_/${commit_date}/' -i "$i"
-          wrapProgram "$i" \
-            --prefix PATH : ${makeBinPath [ pkgs.gnutar pkgs.gzip ]} \
-            --set MINA_LIBP2P_HELPER_PATH ${pkgs.libp2p_helper}/bin/libp2p_helper
-        done
-      '') self.mina-dev.outputs);
+      mina = wrapMina self.mina-dev { };
```

Comment:
Should we consider having 2 attributes? One called `mina` that _isn't_ wrapped. Another called `minaWrapped` that _is_ wrapped. I see this pattern pretty often in nixpkgs. But perhaps it doesn't provide any benefits in this case.

Reply:
There are already two attributes. `mina-dev` is like `mina-unwrapped`, `mina` is like `mina-wrapped`.

Reply:
Ah, I missed that. 👍🏻